styles/RolePlayGateway/template/characters_profile_body.html

```diff
@@ -93,120 +91,75 @@ <h3 style="margin-top:5px;">
         <p style="text-align:justify;">{CHARACTER_HISTORY}</p>
         <!-- ENDIF -->
         <!-- IF S_IS_BETA -->
+        <button onclick="myFunction(stats)">Display Stats</button>
+        <script type="text/javascript">function myFunction(stats) {
```

Comment:
In a future update, use a `<style>` tag for this so JavaScript is not required.

Reply:
Will do! 👍